src/envoy/http/service_control/filter.cc

```diff
@@ -62,15 +62,11 @@ Envoy::Http::FilterHeadersStatus ServiceControlFilter::decodeHeaders(
 
   // Make sure route is calculated
   auto route = decoder_callbacks_->route();
+
+  // This shouldn't happen as the catch-all route match should catch all
+  // the undefined requests.
   if (route == nullptr) {
```

Comment:
````
if (route == nullptr || route->routeEntry() == nullptr) {
```

To match the other filters?

Reply:
This logic is left to the `handler_impl` which checks its nullness when constructed and set the operation.